src/components/menu/MenuHeader.tsx

```diff
@@ -23,8 +25,11 @@ const MenuHeader = ({ children, className, ...rest }: MenuHeaderProps) => {
             role="presentation"
             {...rest}
         >
-            <div className="bdl-MenuHeader-content">{children}</div>
-            <CloseButton onClick={closeMenu} />
+            <div className="bdl-MenuHeader-content">
+                <div className="bdl-MenuHeader-title">{title}</div>
+                <div className="bdl-MenuHeader-subtitle">{subtitle}</div>
```

Comment:
I think it's better to add null/undefined checks for both `title` and `subtitle` here.

Reply:
agreed.